hydromt/models/model_api.py

```diff
@@ -844,17 +913,17 @@ def set_results(self, data, name=None, split_dataset=False):
     ## properties / methods below can be used directly in actual class
 
     @property
-    def crs(self):
-        """Returns coordinate reference system embedded in staticmaps."""
-        return self.staticmaps.raster.crs
+    def crs(self): 
+        """Returns coordinate reference system embedded in staticgeoms."""
+        return self.staticgeoms["region"].crs
 
     def set_crs(self, crs):
         """Embed coordinate reference system staticmaps metadata."""
         return self.staticmaps.raster.set_crs(crs)
 
     @property
-    def dims(self):
-        """Returns spatial dimension names of staticmaps."""
+    def dims(self): #TODO: Right now does not give an error but decide what to do with this
```

Comment:
Add if statements for warning if staticmaps is empty

Reply:
throw a Pending deprecation warning. For now check first if there is data (check xarray is empty) so doesn't bug